02-peer-protocol.md

```diff
@@ -1205,8 +1219,17 @@ A node:
       - SHOULD fail the channel.
 
  A receiving node:
-  - if it supports `option_data_loss_protect`, AND the `option_data_loss_protect`
-  fields are present:
+  - if `option_static_remotekey` applies to the commitment transaction:
+    - if `next_remote_revocation_number` is greater than expected above, AND
+    `your_last_per_commitment_secret` is correct for that
```

Comment:
Not a big deal, but there's a bit of duplication here and below.

Reply:
Indeed.

I decided to do that so eventually when we remove the `option_data_loss_protect` once we're in a world of rainbows and unicorns, the spec will read better and we can just remove the latter paragraph.